lib/restforce/db/cleaner.rb

```diff
@@ -59,7 +59,7 @@ def valid_salesforce_ids
         valid_ids = []
 
         @runner.run(@mapping) do |run|
-          run.salesforce_records { |record| valid_ids << record.id }
+          run.salesforce_instances { |instance| valid_ids << instance.id }
```

Comment:
As I re-read this code, I wonder if `salesforce_instances` could return an `Enumerator`? It seems like there's places where we want to iterate with `each`, places where we want to `map`, places where we want to `inject`...

Anyway, that thought is unrelated to this PR, but it was in my brain.